common/interface.ts

```diff
@@ -1,7 +1,6 @@
 export interface UserInfo {
   userId: string;
   name: string;
-  password: string;
```

Comment:
Here should be changed either:
https://github.com/yioneko/database-course-design-web/blob/077609c763fb4e12eda3122cf01a97279e1a7f80/common/interface.ts#L48

Reply:
Done